python/cuml/__init__.py

```diff
@@ -27,6 +27,13 @@
 from cuml.decomposition.pca import PCA
 from cuml.decomposition.tsvd import TruncatedSVD
 
+from cuml.fil.fil import ForestInference
```

Comment:
```suggestion
```

this is repeated

Reply:
removed the one below it.
This needs to be before ensemble.RF as we call it in RF